db/1.9/modules/ROOT/edits/database-connector-query-single.adoc

```diff
@@ -0,0 +1,215 @@
+== Query a Single Record from a Database Example - Mule 4
+
+The *Query single* operation enables you to execute an SQL query that returns a single record (row) from a database and access the result payload without any additional transformations. This operation does not stream any of the returned fields. When the query is executed, all the values of the returned row are read into memory and returned in the payload. You must be careful, because if your row is too large, you could run out of memory.
+Also, if you use connection pooling you do not have to worry about your application running out of connections, because this operation returns the connection to the pool immediately after the query is executed.
+
+You invoke the *Query single* operation in the same way as you invoke the *Select* operation, although you always get a single result, regardless of the number of records returned by the actual SQL query.
+
+The following example shows how to configure the Anypoint Connector for Database (Database Connector) *Query single* operation in a Mule application that retrieves books information from a table in a MySQL database. +
+
+To accomplish this example, you must: +
+
+* Run an SQL statement that creates a table called `BOOK` and inserts seven records of book information (title, author, bookshopName, isbn, price).
+* Configure the SQL database connection.
+* Create the Mule application.
+* Run, and test the application with curl commands.
```

Comment:
```suggestion
* Run the application and then test it by using with curl commands.
```

Reply:
Also, note that this list item and the one previous are combined in the subtopic that comes later....